createWebNavigator.js

```diff
@@ -1,39 +1,38 @@
 import _ from 'underscore';
 import * as React from 'react';
-import {View, Pressable} from 'react-native';
+import {View, Dimensions} from 'react-native';
 import {useNavigationBuilder, createNavigatorFactory, StackRouter} from '@react-navigation/native';
+import RHPContainer from './components/RHPContainer';
+
+const isSmallScreenWidth = Dimensions.get('window').width <= 800;
```

Comment:
I'm curious how we would plan to make this dynamic if we are declaring the value when the navigator is declared. I think you'd have to refresh the page. Do we have any solution for that yet?